test/generation_test_setup/dummy_page_loader_element.dart

```diff
@@ -186,7 +188,7 @@ class DummyPageLoaderElement implements PageLoaderElement {
   PageLoaderAttributes get properties => throw 'not implemented';
 
   @override
-  PageLoaderAttributes get computedStyle => throw 'not implemented';
+  PageLoaderAttributes get computedStyle => _computedStyle;
```

Comment:
Just make this a final field? No need to indirect through a property

Reply:
I don't think that we have an Effective Dart guideline regarding this. Maybe we should? cc @munificent @kwalrath

Reply:
Sure. I think the guideline would be awesome. 

I decided to use a getter because that's being done for 'classes' getter as well and wanted to be consistent. I'll change that as well.